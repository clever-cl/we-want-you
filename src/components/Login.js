import React from 'react'
import {loginUrl} from '../helpers/spotify'
import "../styles/Login.css"
const Login = () => {
    return (
        <div className="login">
            <a href={loginUrl}> Login with spotify</a>
        </div>
    )
}

export default Login
