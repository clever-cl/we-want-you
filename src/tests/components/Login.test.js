import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme';
import Login from '../../components/Login';

describe('Test in <Login />', () => {
  test('should show component correct', () => {
    const wrapper = shallow(<Login />)
    expect(wrapper).toMatchSnapshot()
  })


})