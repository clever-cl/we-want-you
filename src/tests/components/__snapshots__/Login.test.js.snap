// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <Login /> should show component correct 1`] = `
<div
  className="login"
>
  <a
    href="https://accounts.spotify.com/authorize?client_id=5a867881fb00440cba8f2da627f4f3f5&redirect_uri=http://localhost:3000/&scope=user-read-currently-playing%20user-read-recently-played%20user-read-playback-state%20user-top-read%20user-modify-playback-state&response_type=token&show_dialog=true"
  >
     Login with spotify
  </a>
</div>
`;
